examples/HelloNtpClock/HelloNtpClock.ino

```diff
@@ -4,7 +4,8 @@ using namespace ace_time;
 using namespace ace_time::clock;
 
 static BasicZoneProcessor parisProcessor;
-NtpClock ntpClock;
+NtpClock ntpClock, ntpClock2;
+acetime_t nowSeconds;
```

Comment:
nit: I'd move this into the `setup()` since it is used only there

Reply:
Yes. It's a leftover from the version which used it in loop() as well.